hack/cluster-api/cluster.x-k8s.io_clusterclasses.yaml

```diff
@@ -0,0 +1,385 @@
+
```

Comment:
Since we're going to a separate branch I think we should check them in based on what is in thirdparty but we could document that they need to be generated prior to adding.